### PR TITLE
refactored the FAQ button on nav using tailwind and next/link

### PR DESCRIPTION
**Pull request by BBtxt**
I've refactored the nav button using Tailwind. I started with this Nav button since some styles will be off when we refactor the RoundDateDisplay Icon while moving to Tailwind. I tried to keep the Drop shadow on hover as close as possible to how chakra UI styled their button. I also incorporated Next/Link to get some sweet SEO and prefetching!

### CODE REVIEW

File: components/NavButtons/FAQ.tsx
Comment (nspilman) [inline components/NavButtons/FAQ.tsx:6]: You don't want to use a div here - you want to use a `button`. [Button vs Divs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role). Button to Div makes it less accessible.

File: components/NavButtons/FAQ.tsx
Comment (nspilman) [inline components/NavButtons/FAQ.tsx:11]: remove

**Review by nspilman — CHANGES_REQUESTED** [review]
Eyo - thank you for tackling this! 
This is might be a trickier PR than you might think - maybe not this one, but the next one. 

1. The FAQ hover affect is definitely a little different. Please use the styles in [this Button](https://github.com/nspilman/eptss-site/blob/081e54673c8ef90c51d627daa4d5a3f64e95bb28/styles/Button.ts) styles declaration. 
2. Speaking of [Buttons](https://github.com/nspilman/eptss-site/blob/081e54673c8ef90c51d627daa4d5a3f64e95bb28/styles/Button.ts) - in your next PR, please create a new `Button` component with this FAQ style, and replace all current Charka `Buttons` with this new button, and then delete the Chakra Button style declaration. 

I'm good to approve this once the styles match.

**Comment by BBtxt** [comment]
Sounds good I'll jump into the docs and freak the styling to match Chakra Ui. I was thinking of putting the styles in the tailwind config. So the component would look cleaner

**Comment by nspilman** [comment]
> Sounds good I'll jump into the docs and freak the styling to match Chakra Ui. I was thinking of putting the styles in the tailwind config. So the component would look cleaner

@BBtxt sounds good to me. The feels like a follow up PR item 👍🏻

**Comment by BBtxt** [comment]
I was able to achieve the same glow effect and made sure the styles were similar to Chaka Ui. I used chakra - yellow -300 for the color. I think later we should change this and stick to the Tailwind colors.

**Comment by nspilman** [comment]
Almost there. The color is still off. 
Should be the equivalent of `chakra-yellow-300`

File: components/NavButtons/FAQ.tsx
Comment (nspilman) [inline components/NavButtons/FAQ.tsx:6]: u can yeet `hover:border-yellow-200` now, yeah?

**Review by nspilman — APPROVED** [review]
LGTM - tho please remove the `yellow-200` reference before landing

**Pull request merged**
